### ISSUE TITLE
Blazer, how to use

### ISSUE THREAD

**Issue by sabrogden** [sergey-su/logjoint#39]
I see in the latest you have lots of changes related to Blazer/wasm.  How would you use this? Does this mean it works in the browser?  

When compiling locally I get the error "The working directory "C:\Users\<user>\Desktop\logjoint-master\logjoint-master\trunk\platforms\blazor\logjoint.wasm\../../../../../logjoint-google-plugin/plugin" does not exist.  I don't see any files/folders named logjoint-google-plugin

**Comment by sergey-su** [comment]
Yes, platforms\blazor\logjoint.wasm is an attempt to create browser-only port that could be deployed as a static site (no server code). It is work-in-progress.

Generally it works, but there are problems that make the general purpose deployment impractical at the moment:
- Performance issues related UI freezes. For example, when searching all occurences over a large file. Blazor multithreading support should fix it. See https://github.com/dotnet/runtime/issues/85592.
- User-defined formats require logjoint to compile some dynamically generared code. Support for that was limited in blazor when I checked last time.
- Much inferior single-core performance of blazor compared to the full dotnet. It's due to .net bytecode interpretation instead of JIT. To circumvent that, all the code, including the generated code for user-defined formats and user-provided regexps, should be compiled into wasm byte code in advance. It's called ahead-of-time complition, AOT. This means that an efficient web version needs to know all the user-provided formats at compile time.
